### PR TITLE
Fix token generation when identity is passed via POST

### PR DESCRIPTION
**Pull request by jefflinwood**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -54,18 +54,31 @@ def config():
...
-def token(identity = None):
+@app.route('/token', methods=['GET'])
+def randomToken():
+    return generateToken(fake.user_name())
+
+
+@app.route('/token', methods=['POST'])
+def createToken():
+    # Get the request json or form data
+    content = request.get_json() if request.get_json() else request.form
```
Comment (dprothero) [inline app.py]: I think this could be: `content = request.get_json() or request.form`

File: app.py [hunk 2]
```diff
@@ -54,18 +54,31 @@ def config():
...
+    # get the identity from the request, or make one up
+    identity = content['identity'] if content['identity'] else fake.user_name()
```
Comment (dprothero) [inline app.py]: `identity = content.get('identity', fake.user_name())`

File: app.py (hunk 1 above)
Comment (jefflinwood) [reply app.py]: Fixed in a5e8a25
Thanks!

File: app.py (hunk 2 above)
Comment (jefflinwood) [reply app.py]: Also fixed in a5e8a25
Thanks again!

**Pull request merged**